ioym/src/main.rs

```diff
@@ -117,7 +117,7 @@ where
             let available = match r.fill_buf() {
                 Ok(n) => n,
                 Err(ref e) if e.kind() == std::io::ErrorKind::Interrupted => continue,
-                Err(e) => Err(e)?,
+                Err(e) => return Err(e.into()),
```

Comment:
Why? The advantage of `?` is that it calls `into()`.

Reply:
Indeed, though the disadvantage is having a failed pipeline:
https://rust-lang.github.io/rust-clippy/master/index.html#try_err